test/ToucanRegenBridge.js

```diff
@@ -33,15 +35,17 @@ describe("Bridge contract", function () {
 	});
 
 	beforeEach(async function () {
-		[admin, bridgeAdmin, broker] = await ethers.getSigners();
+		[admin, tokenIssuer, broker] = await ethers.getSigners();
 
 		// Deploy ToucanRegenBridge
-		bridge = await deployBridge(bridgeAdmin.address, nctPool.address);
+		bridge = await deployBridge(admin, nctPool.address);
+		await bridge.connect(admin).grantRole(keccak256(toUtf8Bytes("TOKEN_ISSUER_ROLE")), tokenIssuer.address);
```

Comment:
Near the top of the file:

```typescript
    import { utils } from 'ethers';
```

and then

```suggestion
		await bridge.connect(admin).grantRole(utils.id("TOKEN_ISSUER_ROLE"), tokenIssuer.address);
```

although I'm pretty sure you can also do:

```typescript
		const issuerRole = await bridge.TOKEN_ISSUER_ROLE();
```